examples/kms/vault/csi-kms-connection-details.yaml

```diff
@@ -0,0 +1,45 @@
+#
+# csi-kms-connection-details is an alternative option to configure KMS
+# providers for encrypted volume support.
+# This ConfigMap can be located in the Kubernetes Namespace where Ceph-CSI is
+# deployed. In case the ceph-csi-encryption-kms-config which provides a
```

Comment:
@nixpanic why we have fallback configmap ? In which scenarios it helps?

Reply:
#1793 is still open and should describe the introduction of the ConfigMap. There was no example included with the implementation, making it difficult for users to understand how it works. This example now includes different KMS descriptions as an example.